src/pages/components/subscriptionSection/PlanCard.js

```diff
@@ -3,16 +3,17 @@ import CheckCirledIcon from '../icons/CheckCircledIcon';
 import CloseCirledIcon from '../icons/CloseCircledIcon';
 
 const PlanCard = ({ subscription }) => (
-  <div className='w-[332px] overflow-hidden bg-orange-100 shadow-lg'>
-    <div className='rounded-t-lg bg-orange-900 py-4 text-center text-2xl font-semibold text-white'>
+  <div className='w-[332px] overflow-hidden rounded-lg bg-orange-100 shadow-lg dark:bg-dark-ash-800'>
+    <div className='rounded-t-lg bg-orange-900 py-4 text-center text-2xl font-semibold text-white dark:bg-black'>
       {subscription.name}
     </div>
     <div className='rounded-b-lg'>
-      <ul className='list-none divide-y divide-orange-900 px-9 py-2'>
+      <ul className='list-none divide-y divide-orange-900 px-9 py-2 dark:divide-gray-500'>
         {subscription.content.map((feature, index) => (
           <li className='flex py-4' key={index}>
-            {feature.included ? <CheckCirledIcon /> : <CloseCirledIcon />}{' '}
-            <span className='ml-2'>{feature.description}</span>
+            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'>.</div>
```

Comment:
```suggestion
            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'>.</div>
```
```suggestion
            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'/>
```

Pls remove the tiny dot, so I think you can do it like this or if you have any other idea :bow: